azure_pipelines/pr-validation.yml

```diff
@@ -0,0 +1,81 @@
+# Pipeline will be triggered for PR & any updates on the PR on all branches
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->